disable pytest-flake8 use, it isn't keeping up with flake8 api
see https://github.com/tholo/pytest-flake8/issues/91 etc.

Index: pyproject.toml
--- pyproject.toml.orig
+++ pyproject.toml
@@ -40,14 +40,13 @@ Donation = 'https://opencollective.com/courtbouillon'
 
 [project.optional-dependencies]
 doc = ['sphinx', 'sphinx_rtd_theme']
-# TODO: remove the flake8 dependency when https://github.com/tholo/pytest-flake8/issues/87 is fixed
-test = ['pytest', 'pytest-xdist', 'pytest-flake8', 'pytest-isort', 'pytest-cov', 'coverage[toml]', 'flake8 <5']
+test = ['pytest', 'pytest-xdist', 'pytest-flake8', 'pytest-isort', 'pytest-cov', 'coverage[toml]']
 
 [tool.flit.sdist]
 exclude = ['.*']
 
 [tool.pytest.ini_options]
-addopts = '--isort --flake8 --numprocesses=auto'
+addopts = '--isort --numprocesses=auto'
 
 [tool.coverage.run]
 branch = true
